docs/administration/security/outbound-requests.md

```diff
@@ -66,6 +66,7 @@ In addition, if you also check the box "Help improve Octopus by sending usage st
 | The number of Azure accounts and AWS accounts | 2018.2.2 |
 | Whether you have a custom account configured for the built-in worker | 2018.2.2 |
 | The number of non built-in workers you have | 2018.2.2 |
+| The number of monthly active users that you have | 2019.6 |
```

Comment:
- [ ] Update this version accordingly once https://github.com/OctopusDeploy/octopusdeploy/pull/3808 has shipped